src/nyc_trees/apps/survey/tests.py

```diff
@@ -261,3 +263,68 @@ def _assert_blockface_achievements(self, amount_blockfaces,
         self.assertEqual(amount_achievements, len(achievements))
         achievements = get_achievements_for_user(self.other_user)['achieved']
         self.assertEqual(amount_achievements, len(achievements))
+
+
+class TilerTests(SurveyTestCase):
+    def _exec_tiler_query(self, filename, **kwargs):
+        group_id = kwargs.get('group_id', '0')
+
+        # The path to the tiler SQL queries is defined as a synced
+        # folder in the Vagrant file.
+        path = os.path.join('/opt/tiler/sql/', filename)
+
+        # All tiler queries are in the form "(...) AS query" which isn't
+        # a valid query. We need to prepend a select to this subquery
+        # to make it valid.
+        query = 'SELECT * FROM '
+
+        with open(path, 'r') as f:
+            query += f.read()
+
+        # Replace template placeholders.
+        query = query.replace('<%= group_id %>', group_id)
```

Comment:
I'm trying to avoid regex here because in this case I think it would just make things more complicated. Worst case scenario is that the unit tests would fail if there is extra whitespace somewhere.